specification/columns/commitmentdiscountquantity.md

```diff
@@ -0,0 +1,54 @@
+# Commitment Discount Quantity
+
+Commitment Discount Quantity is the amount of a [*commitment discount*](#commitment-discount) purchased or accounted for in *commitment discount* related [*rows*](#glossary:row). The amounts are denominated in [*Commitment Discount Units*](#glossary:commitmentdiscountunit). The aggregated Commitment Discount Quantity across purchase records pertaining to a particular [Commitment Discount ID](#commitmentdiscountid) during its [*term*](#glossary:term) represents the total *Commitment Discount Units* acquired with that commitment discount. For committed usage, the Commitment Discount Quantity is either the number of *Commitment Discount Units* consumed by a [*row*](glossary:#row) that is covered by a *commitment discount* or is the unused portion of a *commitment discount* over a *charge period*. The CommitmentDiscountQuantity column only applies to *commitment discounts* and not [*negotiated discounts*](#glossary:negotiated-discount). Commitment Discount Quantity is commonly used in commitment discount analysis and optimization use cases.
+
+For usage-based *commitment discounts* (i.e. when *CommitmentDiscountCategory* is "Usage"), the Commitment Discount Quantity reflects the amount of predefined usage purchased or consumed. If *commitment flexibility* is applicable, this value may be further transformed based on additional, provider-specific requirements. For spend-based *commitment discounts*, the Commitment Discount Quantity reflects the prefined amount of spend purchased or consumed.
```

Comment:
```suggestion
For usage-based *commitment discounts* (i.e. when *CommitmentDiscountCategory* is "Usage"), the Commitment Discount Quantity reflects the amount of the commitment discount purchased or consumed. If *commitment flexibility* is applicable, this value may be further transformed based on provider-specific requirements. For spend-based *commitment discounts*, the Commitment Discount Quantity reflects the monetary amount of the *commitment discount* purchased or consumed.
```

Reply:
"predefined usage" and "predefined spend" help parallel the same definitions supplied by `CommitmentDiscountCategory` allow values, so I'd support keeping this consistency.